.travis.yml

```diff
@@ -31,7 +31,7 @@ cache:
 env:
   global:
     - PATH="$HOME/.local/bin:$PATH"
-    - SYMFONY_DEPRECATIONS_HELPER=weak
+    - SYMFONY_DEPRECATIONS_HELPER=weak_vendors
```

Comment:
this one is required? or shouldn't that excluder other deprecations?

Reply:
Ping @greg0ire

Reply:
Revert to `weak`, there is a bug with `weak_vendors` apparently.